scripts/validation/wheel_install_smoke.sh

```
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+WHEEL_GLOB="${1:-dist/*.whl}"
+BOOTSTRAP_DEPS="${ROBOT_SF_WHEEL_INSTALL_SMOKE_DEPS:-loguru numba matplotlib}"
+REPORT_FILE="output/validation/wheel_install_smoke_report.json"
+
+if [[ -z "${1:-}" && "${WHEEL_GLOB}" == "dist/*.whl" ]]; then
+  WHEEL_GLOB="$(ls -1t dist/*.whl 2>/dev/null | head -n 1 || true)"
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)